src/utils.js

```diff
@@ -0,0 +1,200 @@
+class Task {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
+
+function restoreOriginalIcons() {
```

Comment:
Great work Alejandro! I like the solutions you applied; the ES6 syntax is very well done.
What do you think of refactoring your functions into arrow functions? That way you can keep with that ES6 syntax
ex: const restoreOriginalIcons = () => {//your code}